resources/js/src/app/components/itemList/filter/ItemFilterPrice.js

```diff
@@ -47,12 +51,18 @@ Vue.component("item-filter-price", {
         triggerFilter() {
             if (!this.isDisabled) {
                 let facetValue = {
-                    min: this.priceMin,
-                    max: this.priceMax ? this.priceMax : Number.MAX_SAFE_INTEGER
+                    min: this.priceMin ? this.priceMin : 0,
+                    max: this.priceMax ? this.priceMax : this.getMaxPrice()
                 };
 
                 this.updateSelectedFilters(this.facet.id, facetValue);
             }
+        },
+
+        getMaxPrice() {
+            let maxPrice = this.facet.values[this.facet.values.length -1].name.split(' - ')[1];
+
+            return maxPrice ? maxPrice : Number.MAX_SAFE_INTEGER;
```

Comment:
Would a `maxPrice` of `0` therefore evaluate to `Number.MAX_SAFE_INTEGER`?

Reply:
Yes, if max price is equal to 0, then Number.MAX_SAFE_INTEGER will be used.